locales/en/email-strings.ftl

```diff
@@ -0,0 +1,79 @@
+
+click-to-verify = 
+  Select the Verify My Email button within 24 hours to confirm your Firefox Monitor account. 
+  Your report will then be on its way.
+verify-my-email = Verify My Email
+
+scan-another-email = Scan another email in {-product-name}.
+
+automated-message = 
+  This is an automated email; if you received it in error, no action is required.
+
+# Without HTML markup reads: We sent this message to [user’s email address] because the email address opted into alerts from Firefox Monitor.
+we-sent-this-alert = We sent this message to { $userEmail } because the email address opted into alerts from {-product-name}.
+
+unsubscribe = unsubscribe
+
+# This string is followed by an Unsubscribe button, and appears in full: If you no longer want Firefox Monitor alerts, unsubscribe.
+unsubscribe-blurb = 
+  If you no longer want {-product-name} alerts,
```

Comment:
This doesn't work, concatenation is always bad when it comes to localization, because you're imposing the English grammar structure to all other languages. It should use a placeholder instead, with a note explaining how it's used:

```
# $unsubscribe is replaced by a link, using unsubscribe-link as text
If you no longer want {-product-name} alerts, { $unsuscribe }
``` 

And then let each locale move it around as needed.

Reply:
Note: I'm merging https://github.com/mozilla/blurts-server/pull/539 so it doesn't bit-rot but when we fix this concat order we should similarly fix `hibp-attribution` in `app.ftl` as @pdehaan suggests.

Reply:
Though I see in https://github.com/mozilla/blurts-server/pull/539#pullrequestreview-166374782 @flodolo suggests we can use a comment to explain the grammar flow to translators, which @lesleyjanenorton did.

So, if we can't avoid all concatentation in (our use of) handlebars, we should at least make sure to include comments for translators.

Reply:
> Though I see in [#539 (review)](https://github.com/mozilla/blurts-server/pull/539#pullrequestreview-166374782) @flodolo suggests we can use a comment to explain the grammar flow to translators, which @lesleyjanenorton did.

Right, that would be the last resort. But I also have no idea if technically you can have a placeholder in the string, and replace it with markup later.